### ISSUE TITLE
ctrl + cmd + m shortcut doesn't work anymore

### ISSUE THREAD

**Issue by simonemazzoni** [kmikiy/SpotMenu#162]
### Description

The ctrl + cmd + m shortcut to show current song title toast doesn't work anymore since 1.9.0 version. It used to work on previous version.

### Steps to Reproduce

1. Type the shortcut --> nothing happen (it also possibly conflicts with other apps like google chrome for example).

**Expected behavior:** I expect this to work show the toast as before

**Actual behavior:** Nothing is shown

**Reproduces how often:** It happens all the time

### Versions

SpotMenu version: 1.9.0

You can get this information by highlighting SpotMenu.app (most likely in your Applications folder) in finder and pressing space.

MacOS version: macOS sierra 10.12.6 (16G1710)
You can get this information by clicking the  logo on the top left of your menu bar and selecting About This Mac.

### Additional Information

Any additional information, configuration or data that might be necessary to reproduce the issue.

**Comment by kmikiy** [comment]
hey 👋🏻 I think I forgot to mention that i updated it to `ctrl + shift + m`

**Comment by simonemazzoni** [comment]
Ok, that explains 😄 
Please, consider to update the preference menu and the documentation also.

**Comment by kmikiy** [comment]
yeah, I totally forgot, the change wasn't intended to go public 😃

**Comment by mattrossman** [comment]
Any chance we could have to option to select the mapping ourselves?

**Comment by leeyu0911** [comment]
How about add next song hot key?

**Comment by rougiain** [comment]
Love/Dislike shortcuts

**Comment by kmikiy** [comment]
closing because I'm rewriting the app from scratch https://github.com/kmikiy/SpotMenu/issues/279#issuecomment-2987565341
